tests/periph/fmc/main.c

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (C) 2023 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       Test for memories connected to the STM32 FMC/FSMC peripheral
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "benchmark.h"
+#include "od.h"
+#include "periph_conf.h"
+
+#ifndef FMC_BANK
+#define FMC_BANK    0
+#endif
+
+#ifndef MEM_BUFSIZ
```

Comment:
Shouldn't it rather be called:
```suggestion
#ifndef MEM_BUFSIZE
```
?

Reply:
I have used this name in reference to the standard macro `BUFSIZ`.